api-report/container-definitions.api.md

```diff
@@ -453,6 +453,7 @@ export interface IRuntime extends IDisposable {
     request(request: IRequest): Promise<IResponse>;
     setAttachState(attachState: AttachState.Attaching | AttachState.Attached): void;
     setConnectionState(connected: boolean, clientId?: string): any;
+    shutdownPendingBlobs?(): unknown;
```

Comment:
I was hesitant about this as well. I wanted to avoid getPendingLocalState to be async as well but it shouldn't matter since we can resolve instantly if we're not closing